### ISSUE TITLE
SDK 52 Android error

### ISSUE THREAD

**Issue by jpudysz** [jpudysz/react-native-unistyles#321]
### Description

This issue has been reported on Discord, and was verified by me.
With Expo SDK 52.0.0-preview.3 Android is not detecting Unistyles (it looks like it wasn't linked):

![image](https://github.com/user-attachments/assets/8ca59b94-f803-45c8-8da4-8bb8479ad76e)

But works totally fine with Bare RN 0.76:

![image](https://github.com/user-attachments/assets/720275a6-928f-4655-99af-c4604836a93d)

I guess for now there is no action needed, and I will await stable release.

### Steps to reproduce

1. `npx create-expo-app@latest --template default@beta`
2. `yarn add react-native-unistyles`
3. `yarn expo prebuild`
4. Add init config for unistyles and convert one StyleSheet to use theme

### Unistyles version

2.10.0

### React Native version

0.76.0

### Platforms

Android

### Engine

Hermes

### Architecture

Fabric (new)

**Comment by advancedtw** [comment]
apparently your package doesn't have am android manifest file.
simply creating a file in android/src/main/java/com/unistyles/AndroidManifest.xml with this content fixes the problem
```xml

<manifest xmlns:android="http://schemas.android.com/apk/res/android"
          package="com.unistyles">

</manifest>
```

**Comment by jpudysz** [comment]
Closing as it works fine with `52.0.0-preview.7` 🎊
